include/frozen/bits/pmh.h

```diff
@@ -96,13 +96,15 @@ pmh_buckets<M> constexpr make_pmh_buckets(const carray<Item, N> & items,
   result_t result{};
   // Continue until all items are placed without exceeding bucket_max
   while (1) {
-    for (auto & b : result.buckets) {
-      b.clear();
-    }
     result.seed = prg();
     for (std::size_t i = 0; i < N; ++i) {
       auto & bucket = result.buckets[hash(key(items[i]), static_cast<size_t>(result.seed)) % M];
-      if (bucket.size() >= result_t::bucket_max) { continue; }
+      if (bucket.size() >= result_t::bucket_max) {
+        continue;
```

Comment:
The for loop after the continue will never be executed.  Not sure if that was intentional or not.

Reply:
mmmh, whaterver the answer, that's unrelated to that PR, right? :-)

Reply:
wow my bad, you're 100% correct, I'll fix that.